omod/src/main/webapp/fragments/patientdashboard/visits.gsp

```diff
@@ -57,12 +73,39 @@
             encounterCount = ${ param.encounterCount }; // This variable is defined in patientdashboard/patientDashboard.gsp
         <% } %>
         loadTemplates(visitId, ${ patient.id }, fromEncounter, encounterCount);
+
+        <%
+            def visits = patient.allVisitsUsingWrappers;
+            visits.eachWithIndex{ wrapper, idx ->
+                 if(wrapper.getVisit().visitId == param.visitId ||
+                    wrapper.getVisit().visitId == activeVisit.visit.id){
+                    if(wrapper.hasEncounters()){ %>
+                        jq('#i-toggle').show();
+                    <%}
+                 }
+            }
+        %>
     });
+
+    function toggle(){
```

Comment:
This should go in an external js file, and it needs a more descriptive name than just "toggle"